PWGCF/TwoParticleCorrelations/DataModel/TwoParticleCorrelationsFiltered.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef O2_ANALYSIS_TWOPARTFILTERED_H
+#define O2_ANALYSIS_TWOPARTFILTERED_H
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+#include "TwoParticleCorrelationsSkimmed.h"
+
+namespace o2
+{
+namespace aod
+{
+/* we have to change from int to bool when bool columns work properly */
+namespace twopfilter
+{
+DECLARE_SOA_COLUMN(TwoPCollisionCentMult, centmult, float); //! The centrality/multiplicity pecentile
+} // namespace twopfilter
+DECLARE_SOA_TABLE(TwoPAcceptedCollisions, "AOD", "TWOPACCCOLL", //! Accepted reconstructed collisions/events filtered table
```

Comment:
Attention here! The first and third argument must be alike (apart the `s`) to allow merging of DFs in the aod merger

Reply:
Oh! I didn't know it
I will fix it!

Reply:
Done!